src/site/content/en/blog/intersectionobserver-v2/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Trust is Good, Observation is Better—Intersection Observer v2
+subhead: |
+  Intersection Observer v2 adds the capability to not only observe intersections per se, but to also
```

Comment:
Is "v2" how this is styled on the spec itself? I am unable to find the actual spec.

Reply:
It's indeed used like that: https://w3c.github.io/IntersectionObserver/v2/#ref-for-intersectionobserver⑤:~:text=V2.